client/public/css/home.css

```diff
@@ -527,3 +527,92 @@ New color scheme reference is in utils.css
   text-align: center;
   background: rgba(49, 49, 49, 1);
 }
+
+/*--------------------------------------------------------------
+    Login Modal
+--------------------------------------------------------------*/
+.modal.login .modal-header,
+.modal.login .modal-footer {
+  border: none;
+}
+
+.modal.login .modal-content {
+  border-radius: 20px;
+}
+
+.modal.login h4 {
+  text-align: center;
+  margin-top: 20px;
+  margin-bottom: 0;
+  width: 100%;
+}
+
+.modal.login .wrap-input {
+  margin-bottom: 10px;
+}
+
+.modal.login label {
+  margin-bottom: 0;
+}
+
+.modal.login input {
+  width: 100%;
+  border: 2px solid rgba(0, 161, 154, 1);
+  font-size: 1.5rem;
+  line-height: 3rem;
+}
+
+.modal.login .forgot-container {
+  text-align: right;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.modal.login a {
+  color: rgba(0, 161, 154, 1) !important;
```

Comment:
Please avoid important if possible, use a more specific selector in that case

Reply:
There's already an important coming from utils.css `color: inherit !important;`

I can't even override with an id selector unless I use important

Reply:
Ok ya just checked it, well in that case than it's ok, but for the future we should avoid them :D